### PR TITLE
Fixed Edit Button for Location Dashboard

### PR DESCRIPTION
**Pull request by jchaing**
# Description

Fixed the Edit Button on Location Dashboard by pulling in the location state from Redux store

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: plza-fe/src/components/locations/LocationEdit.js [hunk 1]
```diff
@@ -23,10 +25,12 @@ export default function LocationEdit() {
       );
   };
 
+  // console.log(location);
```
Comment (Jeffreyo3) [inline plza-fe/src/components/locations/LocationEdit.js:28]: Looks like we got a stray console.log to remove 😬

File: plza-fe/src/components/locations/LocationEdit.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
 import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
 import { useParams, useHistory } from "react-router-dom";
 import { Message } from "semantic-ui-react";
 import { Form, Input, Dropdown, TextArea, Button } from "formik-semantic-ui";
-import { curr_location } from "../../utils/auth";
+// import { curr_location } from "../../utils/auth";
```
Comment (agyin3) [inline plza-fe/src/components/locations/LocationEdit.js:6]: `curr_location` no longer needed with redux refactor

File: plza-fe/src/components/locations/LocationEdit.js (hunk 1 above)
Comment (jchaing) [reply plza-fe/src/components/locations/LocationEdit.js]: Good catch, removed console.log https://github.com/Lambda-School-Labs/lets-get-pizza-fe/pull/71/commits/701b5ac4ec216b0e13ca4988e06c7ba12f922e60

File: plza-fe/src/components/locations/LocationEdit.js (hunk 2 above)
Comment (jchaing) [reply plza-fe/src/components/locations/LocationEdit.js]: Good catch, removed curr_location https://github.com/Lambda-School-Labs/lets-get-pizza-fe/pull/71/commits/701b5ac4ec216b0e13ca4988e06c7ba12f922e60

**Comment by jchaing** [comment]
[Fix Location's Dashboard Edit Button to view update form](https://trello.com/c/Dd33WMIi/75-fix-locations-dashboard-edit-button-to-view-update-form)

**Pull request merged**